packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -70,3 +70,34 @@ LoadingButton.args = {
     icon: Icon,
     isLoading: true,
 };
+
+export const Rounded = ButtonTemplateWithChild.bind({});
+Rounded.parameters = {
+    design: {
+        type: 'figma',
+        url: 'https://www.figma.com/file/46dEAasj1iEtrVQOwmMswB/00--%3E-Themes-Design-System?type=design&node-id=1128-39276&mode=dev',
```

Comment:
`Design` tab is not present in Storybook, maybe some config is set up incorrectly? Could you check please?

Reply:
Fixed this